src/mode.js

```diff
@@ -14,6 +14,14 @@
  * limitations under the License.
  */
 
+import {urls} from './config';
+
+
+/** @type {string} */
+const thirdPartyFrameHost = urls.thirdPartyFrameHost;
+
+/** @type {boolean} */
+const urlsForceLocalDev = !!urls.localDev;
```

Comment:
@cramforce this increases file sizes a little bit, hoping it goes away with the mode optimization.